packages/block-editor/src/components/block-list/block.native.js

```diff
@@ -432,7 +435,7 @@ const applyWithSelect = withSelect( ( select, { clientId, rootClientId } ) => {
 	// This function should never be called when a block is not present in
 	// the state. It happens now because the order in withSelect rendering
 	// is not correct.
-	const { name, attributes, isValid } = block || {};
```

Comment:
For mobile, there's not really a need to keep passing attributes in the block object because mobile is not extensible by third party plugins.

Reply:
I don't understand this because I see that the wrapped `BlockListBlock` component uses the `attributes` prop several times. For example, when calling `getAccessibilityLabel`. And that uses, for example, `attributes.level` to say "this is a level 2 heading".

Reply:
`attributes` yes, but not `block.attributes`?

Reply:
I see, you're assigning the `attributes` variable elsewhere, with `getBlockAttributes` :) Nevermind.